L1Trigger/Configuration/python/customiseSettings.py

```diff
@@ -1,6 +1,50 @@
 import os.path
 import FWCore.ParameterSet.Config as cms
 
+def L1TSettingsToCaloStage2Params_2017_v1_9_inconsistent(process):
```

Comment:
seems time to implement a helper function? eg
def L1TSettingsToCaloStage2Params(process,str):
   process.load("L1Trigger.L1TCalorimeter.caloStage2Params_"+str+"_cfi")
  return process

then the user calls
process=L1TSettingsToCaloStage2Params(process,"2017_v1_9_inconsistent")